scripts/stac/imagery/collection.py

```diff
@@ -59,6 +63,32 @@ def __init__(
 
         self.add_providers(providers)
 
+    def add_capture_area(self, path: str) -> None:
+        """Add the capture area of the Collection. The `path` of the capture-area.geojson is only used to get its checksum.
+        The `href` or path of the capture-area.geojson is always set as the relative `./capture-area.geojson`
+
+        Args:
+            path: of the capture-area-geojson file
+        """
+        if not self.stac.get("asset"):
+            self.stac["assets"] = {}
+
+        if not self.stac.get("stac_extensions"):
+            self.stac["stac_extensions"] = []
+
+        if StacExtensions.file.value not in self.stac["stac_extensions"]:
+            self.stac["stac_extensions"].append(StacExtensions.file.value)
+
+        file_checksum = checksum.multihash_as_hex(path)
+        capture_area = {
+            "href": f"./{CAPTURE_AREA_FILE_NAME}",
+            "title": "Capture area",
+            "type": ContentType.GEOJSON,
+            "roles": ["metadata"],
+            "file:checksum": file_checksum,
+        }
+        self.stac["assets"]["capture_area"] = capture_area
```

Comment:
You can use [this pattern](https://stackoverflow.com/a/17756005/96588) to avoid the `if not self.stac.get("asset"):` above:

```python
>>> stac = {}
>>> stac.setdefault("assets", {})["capture_area"] = "foo"
>>> stac
{'assets': {'capture_area': 'foo'}}
```